src/katt_callbacks.erl

```diff
@@ -241,7 +245,30 @@ http_request( #katt_request{ method = Method
   Hdrs1 = proplists:delete("x-katt-sleep", Hdrs0),
   Hdrs = proplists:delete("x-katt-timeout", Hdrs1),
   timer:sleep(Sleep),
-  lhttpc:request(Url, Method, Hdrs, Body, Timeout, []).
+  %% need to module-prefixed call for meck's case
+  %% http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+  katt_callbacks:http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+
+http_request_now(Url, Method, Hdrs, Body, Timeout, []) ->
```

Comment:
Also maybe just explain that you're doing this because you want to resemble the signature of lhttpc, that has been used before. Btw, I think that fusco retains lhttpc's signature.